code/modules/reagents/reagent_containers.dm

```diff
@@ -349,16 +349,15 @@ var/list/LOGGED_SPLASH_REAGENTS = list(FUEL, THERMITE)
 		if(can_drink(user))
 			reagents.reaction(user, INGEST)
 			spawn(5)
-				if(reagents)
-					if(user.get_stomach())
-						var/datum/organ/internal/stomach/S = user.get_stomach()
-						reagents.trans_to(S.get_reagents(), amount_per_imbibe) // transfer to stomach
-					else if(istype(user, /mob/living/carbon/human))
-						reagents.trans_to(user, amount_per_imbibe / 4) // greatly reduced reagent absorption without a stomach
-					else
-						reagents.trans_to(user, amount_per_imbibe)
+				ingest(user, amount_per_imbibe)
```

Comment:
Also 2 changes that are halfway towards being bug fixes bundled with a feature (although in fairness, the previous stomach PR was merged with 3 open very valid reviews, 2 of them highlighting the 2 bugs this would fix) but only don't qualify because they're technically rewrites to use the new proc
This would fix the bug where species without stomachs also transfer reagents 4 times slower instead of just members of a species with stomachs but who personally doesn't have a stomach